### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by Hyder47** [Overfitter/biobert_embedding#2]
ERROR: No matching distribution found for torch==1.2.0 (from biobert-embedding==0.1.2).
Below are the pytorch versions installed
torch = 1.5.0+cpu
torchvision = 0.6.0+cpu

How do I proceed with the installation.

**Comment by miko-scibite** [comment]
Anyone happen to find a solution to this?

**Comment by zhuochunli** [comment]
Yes, I encountered the same problem, torch==1.2.0 is no longer supported
